main/states/susi_state_machine.py

```diff
@@ -80,6 +87,25 @@ def __init__(self, renderer=None):
         else:
             logger.warning("Susi has the wake button disabled")
             self.wake_button = None
+    
+    def server_checker(self):
+        response_one = None
+        test_params = {
+        'q': 'Hello',
+        'timezoneOffset': int(time.timezone / 60)
+        }
+        while response_one is None:
+            try:
+                response_one = requests.get('{}/susi/chat.json?{}'
+                .format(self.server_url,urlencode(test_params)) ).result()
```

Comment:
Please care the whitespace (coding style)

Reply:
Same here, not changed by the last commit cb23f58